tests/src/FunctionalJavascript/ContactStorageExportTest.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_contact_forms\FunctionalJavascript;
+
+use Drupal\FunctionalJavascriptTests\WebDriverTestBase;
+use Drupal\contact\Entity\ContactForm;
+use Drupal\contact\Entity\Message;
+
+/**
+ * Test Contact Storage Export form, corporate contact forms.
+ */
+class ContactStorageExportTest extends WebDriverTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
```

Comment:
See the modules from the `CorporateContactFormTest`. I think those will be enough as contact and contact_storage are dependencies of oe_contact_forms

Reply:
it seems file and system are needed also for the export test. (strange enough without system the csv file generated does not have the header row and assert fails.